### PR TITLE
Enhance docs about editor's label

### PR DESCRIPTION
**Pull request by Comandeer**
I copied the table from [https://github.com/ckeditor/ckeditor4/issues/2445#issuecomment-1115068142](https://github.com/ckeditor/ckeditor4/issues/2445#issuecomment-1115068142) and slightly reworded the section about labels.

Additionally, I've added a really nice and professional graphic showing editor's areas.

### CODE REVIEW

File: docs/guide/dev/a11y/README.md [hunk 1]
```diff
@@ -87,10 +87,45 @@ The following features support using CKEditor 4 with assistive technologies such
...
+	</thead>
+	<tody>
+		<tr>
+			<th scope="row">Application</th>
+			<td>{@linkapi CKEDITOR.config#applicationTitle}</td>
+			<td>{@linkapi CKEDITOR.editor#applicationTitle}</td>
+			<td>Rich Text Editor, &lt;editor name&gt;</td>
+		</tr>
+		<tr>
+			<th scope="row">Editable area</th>
```
Comment (jacekbogdanski) [inline docs/guide/dev/a11y/README.md:122]:
```suggestion
			<th scope="row">Editable</th>
```

File: docs/guide/dev/a11y/README.md [hunk 2]
```diff
@@ -87,10 +87,45 @@ The following features support using CKEditor 4 with assistive technologies such
 When the user reaches an editor instance, the screen reader announces it with the following text:
 
 ```html
-Rich text editor, editor1, press Alt+0 for help.
+Rich text editor, editor1.
+Editor, editor1, press Alt+0 for help.
 ```
 
-The `editor1` part of the text is simply the {@linkapi CKEDITOR.config#title name that has been assigned to the editor instance} by the developer. This is even more useful when you have more than one editor instance on the same page. The website creator should make sure that editor instance names are meaningful, though, to make it really useful for the users.
+The `editor1` part of the text is simply the {@linkapi CKEDITOR.editor#name editor's name that has been automatically assigned to the editor instance}. Both parts of the label ("Rich text editor, &lt;editor name&gt;" and "Editor, &lt;editor name&gt;") are fully customized by the developer by using their respective configuration variables. This is most useful when you have more than one editor instance on the same page. The website creator should make sure that editor instance names are meaningful, though, to make it really useful for the users.
```
Comment (godai78) [inline docs/guide/dev/a11y/README.md:94]:
```suggestion
The `editor1` part of the text is simply the {@linkapi CKEDITOR.editor#name editor's name that has been automatically assigned to the editor instance}. Both parts of the label ("Rich text editor, &lt;editor name&gt;" and "Editor, &lt;editor name&gt;") are fully customizable by the developer by using their respective configuration variables. This is the most useful when you have more than one editor instance on the same page. The website creator should make sure that editor instance names are meaningful, though, to make it really useful for the users.
```

File: docs/guide/dev/a11y/README.md [hunk 3]
```diff
@@ -87,10 +87,45 @@ The following features support using CKEditor 4 with assistive technologies such
...
+
+Two parts of the label correspond to two main areas of the editor:
```
Comment (godai78) [inline docs/guide/dev/a11y/README.md:96]:
```suggestion
The two parts of the label correspond to the two main areas of the editor:
```

File: docs/guide/dev/a11y/README.md [hunk 4]
```diff
@@ -87,10 +87,45 @@ The following features support using CKEditor 4 with assistive technologies such
...
+
+* the application area – the whole editor, including its toolbars,
+* the editable area – the area where the user types.
+
+{@img assets/img/a11y_labels.png Two main areas of the editor: the application one (whole editor with toolbars included) and the editable one (the text editing area).}
```
Comment (godai78) [inline docs/guide/dev/a11y/README.md:101]:
```suggestion
{@img assets/img/a11y_labels.png Two main areas of the editor: the application one (the whole editor with toolbars included) and the editable one (the text editing area).}
```

**Review by godai78 — APPROVED** [review]

**Review by jacekbogdanski — APPROVED** [review]
Thanks @Comandeer and @godai78!

**Pull request merged**